docs/features.md

```diff
@@ -29,22 +29,24 @@ Example:
 
 #### Since: generic-worker 5.3.0
```

Comment:
This should probably mention the version that ed25519 support was added somewhere.

Reply:
Yeah, I wasn't sure how best to do that. Added another `#### Since:` line with a `TBD`; maybe that will work. We could add another `#### Since:` line when we remove gpg support.